src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -172,6 +172,7 @@ const AccessibilityRequestDetailPage = () => {
       variant="unstyled"
       asCustom={Link}
       to={`/508/requests/${accessibilityRequestId}/documents/new`}
+      data-testid="Upload-New-Document"
```

Comment:
Are we using this casing format for other data-testids?  I do see lowercased-kebab case elsewhere, but not sure if we have consistency.

Reply:
I changed the format.